.idea/libraries/jetty.xml

```diff
@@ -1,7 +1,7 @@
 <component name="libraryTable">
   <library name="jetty">
     <CLASSES>
-      <root url="jar://$PROJECT_DIR$/build/third_party/java/jetty/jetty-repacked.jar!/" />
+      <root url="jar://$PROJECT_DIR$/third_party/java/jetty/jetty-repacked.jar!/" />
```

Comment:
You shouldn't be running clean so much ;) with the switch to buck it actually doesn't make sense to do it